### ISSUE TITLE
Links are not focusable in Safari

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/john-travoltage#204]
Focus can be set programatically but not by pressing tab.

**Comment by jessegreenberg** [comment]
Here is the HTML
```html
<a href="http://phet.colorado.edu/en/licensing/html" target="_blank" id="60-159-376-375-374-371">‪Terms, Privacy &amp; Licensing‬</a>
```

**Comment by jessegreenberg** [comment]
Links needed min width to be visible in Safari, the above commit fixes the problem. Closing.

**Comment by jessegreenberg** [comment]
Reopening, links still aren't focusable in Firefox.

**Comment by jessegreenberg** [comment]
Hmm, not sure what was happening, they are focusable now. Closing
